styles.css

```diff
@@ -0,0 +1,96 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 375px;
+  height: 100%;
+  margin: 0 auto;
+}
+
+header {
+  background-color: #3c3a39;
+  height: 10vh;
+  width: 100%;
+}
+
+ul {
+  display: flex;
+  justify-content: space-between;
+  align-items: end;
+  padding: 25px 30px;
+}
+
+ul li {
+  list-style-type: none;
+}
+
+ul li a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.name {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  font-style: normal;
+  font-family: Arial, sans-serif, 'Crete Round';
+}
+
+.head-line {
+  background-image: url("./Img/Illustration-header.svg");
```

Comment:
[Optional] Great job adding the background image, as suggested by the previous reviewer💪. However, the size of your background image is a bit too big and overlaps with the main content. I strongly suggest you reduce the size of the image so your main content stands out🫡

Your design:
<img width="184" alt="image" src="https://user-images.githubusercontent.com/37116322/219424484-544fe119-9ebb-4d38-aba5-ab75dac99dd9.png">

Figma design:
<img width="184" alt="image" src="https://user-images.githubusercontent.com/37116322/219424722-ae80ab25-161c-4dcb-938c-a1003c17a29d.png">